lib/atom-markdown-image-paste.js

```diff
@@ -41,7 +41,7 @@ export default {
     const imagePath = subDirectoryPath + "/" + imageFilename
     const relativeImagePath = subDirectory + "/" + imageFilename
 
-    if (!fs.existsSync(subDirectoryPath)){
+    if (!fs.existsSync(subDirectoryPath, { recursive: true })){
```

Comment:
Hey @zhanliquan, thanks for opening this PR. I believe the error is being thrown on the next line when `mkdirSync` is called.

Reply:
I know this PR has been sitting for a while, however, if you care to take the time to fit it, I would be happy to merge it and release an updated version to apm.